packages/ds-healthcare-gov/src/components/Accordion/index.tsx

```diff
@@ -1 +1 @@
-export * from './AccordionItem';
+import './AccordionItem';
```

Comment:
Using "import" without any specific things we're trying to import should make it more clear that we need this for its side effects and that the magic isn't in the exports.

Reply:
I think I understand now. By using an import for `AccordionItem`, we emphasize that the primary goal of importing this component here is to apply the healthcare modifications globally via side effects, rather than to reuse it elsewhere in the `ds-healthcare-gov` package.

Reply:
Yeah, exactly